### ISSUE TITLE
Retry connecting to the eventbus if connection could not be established

### ISSUE THREAD

**Issue by TiruYasha** [mikederksen/TeamRemi#19]
We should make a retry functionality if there is no connection. And we should add a option for specifying the retry timer.

**Comment by mikederksen** [comment]
Sounds good. Maybe add it as a parameter to the BusProvider and in turn to the initializer and extension method? I think we should keep the option open for creating the publishers/listeners directly (or via some factory) instead of dependency injection

**Comment by TiruYasha** [comment]
Yes a factory would be nice, but we should make a new issue for that. 
As for the retry parameters. Maybe it is not necessary for the busprovider, because we only have to implement it in the initializer. If the connection fails the Createconnection should throw an exception.

**Comment by mikederksen** [comment]
Makes sense, yes. We can then catch this exception in the initializer or in another place where the busprovider is created
